Startup.cs

```diff
@@ -26,12 +26,12 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddDbContext<VideoContext>(opt =>
```

Comment:
Tu removeu o `VideoContext`. Quando for mergear, ou vai dar conflito ou vai removar essa entidade.

Reply:
Feito :smile: